src/cli/java/org/commcare/util/screen/MultiSelectEntityScreen.java

```diff
@@ -87,6 +87,17 @@ protected void updateSession(CommCareSession session) {
         }
     }
 
+    @Override
+    public boolean referencesContainStep(String stepValue) {
+        try {
+            String[] cachedSelection = entitiesSelectionCache.read(stepValue);
+            return cachedSelection != null;
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
```

Comment:
Does this end up being logged anywhere? I would have though that using a logger would be better.